internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -160,6 +160,64 @@ func resourceMonitorActivityLogAlert() *pluginsdk.Resource {
 							ConflictsWith: []string{"criteria.0.recommendation_category", "criteria.0.recommendation_impact"},
 						},
 						//lintignore:XS003
+						"resource_health": {
+							Type:     pluginsdk.TypeList,
+							Computed: true,
+							Optional: true,
+							Elem: &pluginsdk.Resource{
+								Schema: map[string]*pluginsdk.Schema{
+									"current_health": {
```

Comment:
we can probably remove health as its part of the block name

Reply:
This is resolved